packages/shared/src/types/shared.ts

```diff
@@ -145,6 +145,16 @@ export interface ControlProportionSettings {
 
 export type Stage = 'PROD' | 'CODE' | 'DEV';
 
+/**
+ * Secondary tests are for experimenting with targeting, e.g. measuring the impact of not showing banners on certain sections.
+ * It is not a message test and should not affect what the user sees once they're in a test.
+ * But we do need to carry the test/variant names through in the tracking.
+ */
+export interface SecondaryAbTest {
```

Comment:
not sure about the name here, but I wanted it to be more general as it's going to be in the acquisition/component events.

Reply:
yeah perhaps it should be specific (`TargetingAbTest`), I can't think what else this would be used for.
I worry about putting them both in an array - should there always be a 'primary' test?
Also, we're going to have to work with the existing ophan model: https://github.com/guardian/ophan/blob/main/event-model/src/main/thrift/componentevent.thrift#L241

Reply:
Ahhhh yeah hadn't thought about the ophan model. In that case I guess it's easiest to just add a couple of new fields? Perhaps something like `Secondary`/`Meta` is the way to go then?